### ISSUE TITLE
Styleguide/Specs Tidying

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-ux#53]
- [x] Update styleguide per completed branding & color schema work
   - **Delivered:** https://scene.zeplin.io/project/5c807ea562f734bd2756b243
- [x] Finalize/Deliver left pane branding art
   - **Delivered:** [#17 (comment)](https://github.com/freedomofpress/securedrop-ux/issues/17#issuecomment-478796842)
- [x] Finalize/Spec ID badge styling
  - **Delivered:** https://zpl.io/aRxW3Qe
- [x] Create spec for short error messages 
  - **Delivered:** https://zpl.io/aw4MKlN
- [x] Create spec for username truncation, hover  
  - **Delivered:** https://zpl.io/aRxOBOz
- [x] Update spec for offline mode to final colors/styling  
   - **Delivered**, tagged _Preliminary_ until further determination: https://zpl.io/bA7MRee
   - See [explorations in comment, below](https://github.com/freedomofpress/securedrop-ux/issues/53#issuecomment-490675791), if interested in seeing other directions.
- [x] Update spec for AuthWidget to final colors/styling 
   - **Yep:** https://zpl.io/aRxW3Qe

Also:
- [x] Update typography specs in Zeplin
  - **Yepitty yep**; no more Open Sans: https://scene.zeplin.io/project/5c807ea562f734bd2756b243
- [x] Update _refresh_ icon in GitHub inventory issue
  - **Yep:** https://github.com/freedomofpress/securedrop-ux/issues/17#issuecomment-472165003
- [x] Update spec for "Sign in" from AuthWidget in "Offline" to final colors/styling (grrr...)
   - **Delivered**, tagged _PostBeta_: https://zpl.io/bzqmDME
   - Note: Uncertain if this item was added before or after @eloquence clarified that _any_ code differences whatsoever in the overlay's content from what loads into the initial LogIn window, is outta scope for Beta. Hence, the _PostBeta_ tag on this solution.
- [x] Circulate final agreed upon styleguide for review
   - Ask @creviera. **_Yep!_**

New(ish) aggregate ["Styleguide(ish)" home for spex, in the Client Repo](https://github.com/freedomofpress/securedrop-client/wiki/Styleguide(ish))

**Comment by ninavizz** [comment]
Note: The word "spec" is not exactly defined yet (heh) in the canon of SecureDrop UX. For this Issue I'm qualifying that need having been met, by what I consider to be adequate assets created in Zeplin... and/or the thingy spoken-to in the Wiki Styleguide(ish).

**Comment by ninavizz** [comment]
Hadn't really done any proper "Explorations" of Offline Mode change(s)... so this one's it. https://invis.io/BSRX8TUN8K3#/362071665_Offline_Bars

To fulfill the checkbox req of this Issue, tho, I'll be spec'ing the top-bar portion of [E4](https://invis.io/BSRX8TUN8K3#/362074172_E4) (see above link for more context on exploration). Ideally I'd also like the SD in the logo in the Branding Bar to also dim, but that seems like a lot more work (correct me if I'm wrong, @creviera?). Swapping-out that SVG art for offline mode is ez, I'm happy to generate that art!

**Comment by eloquence** [comment]
Thanks for all the updates! Just a quick note that I'm still seeing some OpenSans references when I go here (just the first random screen I picked from Zeplin):
https://app.zeplin.io/project/5c807ea562f734bd2756b243/screen/5c8094973ff280626252ce31

( "Add | Open" below message, Top bar including "Last Refresh" and Briefcase.) Are all old specs meant to be updated, or am I looking at a non-canonical spec?

**Comment by ninavizz** [comment]
Ugh. This ticket was created to guide me in: 
- Updating all out-of-date things.
- Writing appropriate documentation to guide folks in navigating what's canonical, what's not, and where to go for the right stuff.

Thx for your attention to detail in catching some gaps in both. 😄 

Give 'er a spin, again?

**Comment by eloquence** [comment]
Thanks, appreciate the helpful README slide in Zelpin and the clarification in the styleguide. I think using tags to identify what's canonical definitely makes it easier to circumnavigate outdated info. We can keep tweaking the organization of the tags as we build, esp. interested in ongoing feedback from @creviera on the organization here.

As far as I can tell, there are now just a couple of OpenSanses remaining in the canonical `text-styling` screens:

- Download bar below the message in https://app.zeplin.io/project/5c807ea562f734bd2756b243/screen/5cbf4b26849b5a5407e45f41

- Download all in https://app.zeplin.io/project/5c807ea562f734bd2756b243/screen/5cc25aa44a92851016e0bbf2

**Comment by ninavizz** [comment]
It'd be my preference to simply not have any out-dated info whatsoever... and I'm working towards that. It's a balance, tho, and I don't want to waste precious time where hand-waving and clear `get off my lawn` indicators suffice. Vague and frustrating as those may be, for others. Balance.

- Download bar below the message: Deleted this screen, as it served no further unique purpose un-served by other screens
- Download all: Updated static item w/ library item

Thank you again, Erik, for taking the time and attention to detail, to comb through all of this. I appreciate how tedious it is—and assure that I've already spent many hours and adapted some of my own workflows, to make this cleaner & more efficient. Looking forward to hearing from @creviera how it's shaped-up for her needs, over the past week she's been OOTO at PyCon! #itsaprocess

Pls continue to let me know if you find any other inconsistencies, and I'll get right on 'em!
